pkg/controller/core/workload_controller.go

```diff
@@ -271,7 +271,7 @@ func handlePodOverhead(wl *kueue.Workload, c client.Client) {
 		if pod.Spec.RuntimeClassName != nil && len(pod.Spec.Overhead) == 0 {
 			var runtimeClass nodev1.RuntimeClass
 			if err := c.Get(ctx, types.NamespacedName{Name: *pod.Spec.RuntimeClassName}, &runtimeClass); err != nil {
-				klog.Error(err, "Could not get RuntimeClass")
+				klog.Error("Could not get RuntimeClass: ", err)
```

Comment:
```suggestion
				klog.ErrorS(err, "Could not get RuntimeClass: ")
```

Reply:
Use the log in the WorkloadReconciler object. You can pass it through a context that you can add to the function.